mirrord/cli/src/connection.rs

```diff
@@ -17,6 +17,16 @@ pub(crate) struct AgentConnection {
 }
 
 /// Creates an agent if needed then connects to it.
+///
+/// First it checks if we have an `operator` in the [`config`](LayerConfig), which we do if the
+/// user has installed the mirrord-operator in their cluster, even without a valid license. And
+/// then we create a session with the operator with [`create_operator_session`].
+///
+/// If there is no operator, or the operator session creation failed (e.g. lack of license), then
```

Comment:
I don't think this is true, we use `?` here on result from `OperatorApi::create_session`.

Reply:
Good good